implementations/build-topaz.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e # make script fail on first error
+SCRIPT_PATH=`dirname $0`
+source $SCRIPT_PATH/script.inc
+
+INFO Build Topaz
+pushd $SCRIPT_PATH
+  git clone git://github.com/topazproject/topaz
+  pushd topaz
+    git checkout tim/features
+    pip install invoke
```

Comment:
I needed to update my `invoke` with `pip install --upgrade invoke`, otherwise I got:
```
'travis.install_requirements' did not receive all required positional arguments!
```
Does it make sense to add the `--upgrade` flag here?